tests/www/apply/test_submit.py

```diff
@@ -40,8 +40,16 @@
 
 
 class ApplyTest(TestCase):
+    def test_siae_with_no_members(self):
+        siae = SiaeFactory()
+        user = JobSeekerFactory()
+        self.client.force_login(user)
+        url = reverse("apply:start", kwargs={"siae_pk": siae.pk})
+        response = self.client.get(url)
+        assert response.status_code == 403
```

Comment:
Peut-être aussi tester le texte pour être sûr que c'est bien cette 403 et pas une autre ?
```suggestion
        assert response.status_code == 403
        assertContains(response, "Cet employeur n'est pas inscrit, vous ne pouvez pas déposer de candidatures en ligne.")
```